### DISCUSSION TITLE
A section opition between two RA/DEC coordinates types for AstroCalc

### DISCUSSION THREAD

**Discussion by Ironman65** [Stellarium/stellarium#845] [Q&A] [answered]
This is just an enhancement request. In the upper right hand corner after an object is selected in the information dialog, there’s two sets coordinates of RA/DEC given. RA/DEC (J2000.0) the other RA/ DEC (on date). When you use the positions and ephemeris part of AstroCalc it only displays the RA/DEC (J2000.0) coordinates only! But the positions part allows you to select between RA/DEC and horizontal coordinates by a little box you check in the upper left-hand corner. If possible could a selection box be added to the positions and ephemeris parts to enable you to select between the RA/DEC (J2000.0) and RA/DEC (on date) coordinates. 

Here’s an interesting effect between the two sets of coordinates by example.
First go to LOCATION and then select MARS. Once you’re on the surface of Mars from there turn on the EQUATORIAL GRID [E]. Then click on AstroCalc. Then go to EPHEMERIS and select the celestial body SUN and then calculate ephemeris. You’ll see that the RA/DEC (J2000.0) coordinates don’t match the equatorial grid as seen from Mars. That’s because the RA/DEC (on date) matches the Martian equatorial grid or any other planets surface upon which you are standing. On Earth it’s more accurate because of procession especially as time goes on. Both coordinates are good to compare to see on how far the Earth’s axis wobbles. I know that any new enhancement might cause a program to crash, if this does; just disregard it. Thanks for all you do!

**Comment by alex-w** [comment]
> First go to LOCATION and then select MARS. Once you’re on the surface of Mars from there turn on the EQUATORIAL GRID [E]. Then click on AstroCalc. Then go to EPHEMERIS and select the celestial body SUN and then calculate ephemeris. You’ll see that the RA/DEC (J2000.0) coordinates don’t match the equatorial grid as seen from Mars.

Why you expected to see no difference between coordinates, when you enable equatorial grid for current epoch and compare these coordinates with equatorial coordinates on epoch J2000.0? You may enable equatorial grid for epoch J2000.0.

**Comment by gzotti** [comment, marked as answer]
Coordinates "J2000" are standardized to the orientation of Earth's axis at J2000 and will not adjust to any other planet's axis at J2000. They resemble the standard for today's generation of atlases.
